Python/Temperature_Conversion/Temperature_Conversion.py

```diff
@@ -0,0 +1,67 @@
+""" TEMPERATURE CONVERSION """
+
+def convert(f_scale, temp, to_scale):
+    """
+    to convert from one scale to another
+    Arguments:
+    :f_scale: scale from which the temperature is to be comverted
+    :temp : temperature to be converted
+    :to_scale: scale to which the temperature is to be converted
+
+    :return : string of converted temperature with scale 
+    """
+    converted = temp
+    degree = ""
+
+    # from celsius
+    if(f_scale == 1):
+        degree = "°C"
+        # to Fahrenheit
+        if(to_scale == 2):
+            converted = 9.0/5.0 * temp + 32
+            degree = "°F"
+        # to Kelvin
+        if(to_scale == 3):
+            converted += 273.15
+            degree = "K"
+
+    # from Fahrenheit
+    elif f_scale == 2:
+        degree = "°F"
+        if(to_scale != 2):
+            # to celsius
+            converted = (temp-32)*5.0/9.0
+            degree = "°C"
+            # to kelvin
+            if(to_scale == 3):
+                converted += 273.15
+                degree = "K"
+
+    # from Kelvin
+    else:
+        degree = "K"
+        if(to_scale != 3):
+            # to Celsius
+            converted -= 273.15
+            degree = "°C"
+            # to Fahrenheit
+            if(to_scale == 2):
+                converted = 9.0/5.0 * converted + 32
+                degree = "°F"
+    return str(converted)+degree
+
+'''driver code'''
+def main():
```

Comment:
The docstring looks awesome for the above function, more here please

Reply:
Sure Thanks!